### ISSUE TITLE
How to finetune the reward model itself?

### ISSUE THREAD

**Issue by yzxyzh** [LAION-AI/Open-Assistant#2780]
Hi!

Thanks for your wonderful work! I'm fairly new in chatbot training, and I have encountered a problem trying to replicate your result. Suppose we have already trained a RM, and we have gathered some (maybe quiet little, e.g., 10) bad QA pairs. How can we let our RM model learn this additional message, while keeping its original ability?

Thanks.

**Comment by olliestanley** [comment]
You need to finetune the model using the negative samples as ground truths
